cmd/backup-manager/app/cmd/backup.go

```diff
@@ -46,6 +47,10 @@ func NewBackupCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&bo.TLSClient, "client-tls", false, "Whether client tls is enabled")
 	cmd.Flags().BoolVar(&bo.TLSCluster, "cluster-tls", false, "Whether cluster tls is enabled")
 	cmd.Flags().BoolVar(&bo.SkipClientCA, "skipClientCA", false, "Whether to skip tidb server's certificates validation")
+	cmd.Flags().StringVar(&bo.Mode, "mode", string(v1alpha1.BackupModeSnapshot), "backup mode, which is log or snapshot(default)")
+	cmd.Flags().StringVar(&bo.SubCommand, "subcommand", string(v1alpha1.LogStartCommand), "the log backup subcommand")
```

Comment:
flags usage is not accurate
"backup mode, which is log or snapshot(default)" => "backup mode, which is pitr or snapshot(default)"

Reply:
ok, i think you mean the restore cmd is not accurate in [here](https://github.com/pingcap/tidb-operator/pull/4694/files#diff-acd3036d4ac2e01282ab2bf1035fa449ecaeb9abc5ff8f5583b10e63aabc0d52R52)

Reply:
yes, you are right